[0001].[2021.04.03.02.08.38.630661].SRV1.INF:Main.py.<module>.330:Main Program Started
[0002].[2021.04.03.02.08.38.631662].SRV1.ERR:Main.py.<module>.333:Artificial Error Message for testing purpose
[0003].[2021.04.03.02.08.38.631662].SRV1.INF:Main.py.<module>.334:Command Line iniFileName=default.ini
[0004].[2021.04.03.02.08.38.632661].SRV1.INF:Main.py.<module>.336:Command Line VerboseMode=None
[0005].[2021.04.03.02.08.38.632661].SRV1.INF:Main.py.<module>.339:AppSpoManager.DebugVerboseModeOn=True
[0006].[2021.04.03.02.08.38.633662].SRV1.INF:Main.py.<module>.340:AppSpoManager.DebugInfoMessages=True
[0007].[2021.04.03.02.08.38.633662].SRV1.INF:Main.py.<module>.341:AppSpoManager.DebugErrorMessages=True
[0008].[2021.04.03.02.08.38.634662].SRV1.INF:Main.py.<module>.342:AppSpoManager.DebugTargetConsole=False
[0009].[2021.04.03.02.08.38.634662].SRV1.INF:Main.py.<module>.343:AppSpoManager.DebugTargetFile=True
[0010].[2021.04.03.02.08.38.635663].SRV1.INF:Main.py.<module>.344:AppSpoManager.DebugTargetFilePath=Debugs\Debug1
[0011].[2021.04.03.02.08.38.635663].SRV1.INF:Main.py.<module>.345:AppSpoManager.DebugLinesPerFile=1000
[0012].[2021.04.03.02.08.38.636665].SRV1.INF:Main.py.<module>.346:AppSpoManager.AppRunFolder=c:\Apps\PythonTry1
[0013].[2021.04.03.02.08.38.636665].SRV1.INF:Main.py.<module>.369:ModuleFileNameOnly=Main.py
[0014].[2021.04.03.02.08.38.637666].SRV1.INF:Main.py.<module>.377:File=c:\Apps\PythonTry1\Main.py
[0015].[2021.04.03.02.08.38.637666].SRV1.INF:Main.py.<module>.378:FileDirName=c:\Apps\PythonTry1
[0016].[2021.04.03.02.08.38.638667].SRV1.INF:Main.py.<module>.379:FileAbsPath=c:\Apps\PythonTry1\Main.py
[0017].[2021.04.03.02.08.38.638667].SRV1.INF:Main.py.<module>.382:ModuleName1=__main__
[0018].[2021.04.03.02.08.38.639668].SRV1.INF:Main.py.<module>.383:ModuleName2=Main
[0019].[2021.04.03.02.08.38.639668].SRV1.INF:Main.py.<module>.392:Main Program Started
[0020].[2021.04.03.02.08.38.640668].SRV1.INF:Main.py.<module>.393:Python Executable Path=C:\Program Files\Python39\python.exe
[0021].[2021.04.03.02.08.38.640668].SRV1.INF:Main.py.<module>.394:Python Interpreter Version=3.9.2 (tags/v3.9.2:1a79785, Feb 19 2021, 13:44:55) [MSC v.1928 64 bit (AMD64)]
[0022].[2021.04.03.02.08.38.641669].SRV1.INF:Main.py.<module>.397:Sys.Path=['c:\\Apps\\PythonTry1', 'C:\\Program Files\\Python39\\python39.zip', 'C:\\Program Files\\Python39\\DLLs', 'C:\\Program Files\\Python39\\lib', 'C:\\Program Files\\Python39', 'C:\\Users\\user2\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\Program Files\\Python39\\lib\\site-packages']
[0023].[2021.04.03.02.08.38.670316].SRV1.INF:Main.py.<module>.444:Showing MainWindow
[0024].[2021.04.03.02.08.49.829374].SRV1.INF:Main.py.actionDebugTest1Handler.154:192.168.0.1, KAT1.ODA1, KORÝDOR DUVAR
192.168.0.2, KAT2.ODA2, TAVAN HOPARLÖR
192.168.0.3, KAT3.ODA3, KAPI_ÖNÜ_ANANS
[0025].[2021.04.03.02.08.49.830375].SRV1.INF:Main.py.actionDebugTest1Handler.171:List of All Nodes Read From IniFile------
[0026].[2021.04.03.02.08.49.831376].SRV1.INF:Main.py.actionDebugTest1Handler.173:Node[0] Order=[0] IP=192.168.0.1] Place=]KAT1.ODA1] Desc=[KORÝDOR DUVAR]
[0027].[2021.04.03.02.08.49.833377].SRV1.INF:Main.py.actionDebugTest1Handler.173:Node[1] Order=[1] IP=192.168.0.2] Place=]KAT2.ODA2] Desc=[TAVAN HOPARLÖR]
[0028].[2021.04.03.02.08.49.834378].SRV1.INF:Main.py.actionDebugTest1Handler.173:Node[2] Order=[2] IP=192.168.0.3] Place=]KAT3.ODA3] Desc=[KAPI_ÖNÜ_ANANS]
